### PR TITLE
From participant to task

### PR DESCRIPTION
**Pull request by northox**

### CODE REVIEW

**Comment by northox** [comment]
Relates to #21

**Comment by jfrioux** [comment]
Fix the conflicts first?

**Comment by northox** [comment]
I did not asked to review still.

**Comment by jfrioux** [comment]
All right then!!!!

**Review by jfrioux — APPROVED** [review]

File: app/controllers/floristry/webtask_controller.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Floristry
-  class WebparticipantController < ::ApplicationController
+  class WebTaskController < ::ApplicationController
```
Comment (jfrioux) [inline app/controllers/floristry/webtask_controller.rb:2]: This should be `WebtaskController` and not `WebTaskController`, to fix loading/require issue

**Pull request merged**